chirps/base_app/management/commands/generate_embeddings.py

```diff
@@ -0,0 +1,133 @@
+import json
+import os
+import sys
+
+import cohere
+import numpy as np
+import openai as openai_client
+from django.apps import apps
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    """Custom Django management command for generating embeddings using the specified service and model."""
+
+    help = 'Generate embeddings for the input data'
+
+    def add_arguments(self, parser):
+        """Define and add the required arguments for the command."""
+        parser.add_argument('service', type=str, help='Service to use for embeddings (either "OpenAI" or "cohere")')
```

Comment:
I've addressed this in a different way. Now the user is prompted to select from a list of available services and to provide the number of the service instead of typing the whole name.